pubsub/azure/servicebus/message_test.go

```diff
@@ -121,108 +115,17 @@ func TestNewASBMessageFromPubsubRequest(t *testing.T) {
 				require.NotNil(t, err)
 			} else {
 				require.Nil(t, err)
-				assert.Equal(t, tc.expectedAzServiceBusMessage.Data, msg.Data)
-				assert.Equal(t, tc.expectedAzServiceBusMessage.ID, msg.ID)
+				assert.Equal(t, tc.expectedAzServiceBusMessage.Body, msg.Body)
+				assert.Equal(t, tc.expectedAzServiceBusMessage.MessageID, msg.MessageID)
 				assert.Equal(t, tc.expectedAzServiceBusMessage.CorrelationID, msg.CorrelationID)
 				assert.Equal(t, tc.expectedAzServiceBusMessage.SessionID, msg.SessionID)
 				assert.Equal(t, tc.expectedAzServiceBusMessage.ContentType, msg.ContentType)
 				assert.Equal(t, tc.expectedAzServiceBusMessage.ReplyTo, msg.ReplyTo)
-				assert.Equal(t, tc.expectedAzServiceBusMessage.TTL, msg.TTL)
+				assert.Equal(t, tc.expectedAzServiceBusMessage.TimeToLive, msg.TimeToLive)
 				assert.Equal(t, tc.expectedAzServiceBusMessage.To, msg.To)
-				assert.Equal(t, tc.expectedAzServiceBusMessage.Label, msg.Label)
-				assert.NotNil(t, msg.SystemProperties)
-				assert.Equal(t, tc.expectedAzServiceBusMessage.SystemProperties.PartitionKey, msg.SystemProperties.PartitionKey)
-				assert.Equal(t, tc.expectedAzServiceBusMessage.SystemProperties.ScheduledEnqueueTime.Unix(), msg.SystemProperties.ScheduledEnqueueTime.Unix())
-			}
-		})
-	}
-}
-
-func TestNewPubsubMessageFromAzServiceBusMessage(t *testing.T) {
```

Comment:
This test can't be implemented with track2 SDKs. The method to create these objects is un-exported.

Can be re-added if https://github.com/Azure/azure-sdk-for-go/pull/17848 is merged